src/styles/saas-list.less

```diff
@@ -1,76 +1,104 @@
+.build-applications-view {
+
+  h1 {
+    color: @saas-text;
+    margin: 0;
+    padding: 20px;
+    padding-left: (@grid-gutter-width / 2);
+    width: 100%;
+  }
+}
+
 .saas-offerings-container {
-  background: @color-pf-black-200;
-  padding-left: 0px;
 
   .spinner-container {
     padding-top: 60px;
     padding-bottom: 60px;
   }
 
-  .card-view-pf {
-    padding-left: 21px;
-  }
-
-  .card-view-pf .card-content {
-    background-color: @color-pf-white;
-    cursor: pointer;
-    margin: 0;
+  .saas-list {
+    display: flex;
```

Comment:
Since you don't need flex to achieve your layout for mobile, it's a bit simpler to only do the flex-ing inside of the media query below (line 26).  That way, there is no need to set and reset flex-direction.  In other words, move lines 20, 22, 23 to line 27.  Delete line 21 and 27.